drivers/entropy/entropy_mcux_caam.c

```diff
@@ -30,6 +35,8 @@ static int entropy_mcux_caam_get_entropy(const struct device *dev,
 	caam_handle_t handle;
 	uint16_t read_length = 0;
 	uint16_t insert_idx = 0;
+	int ret = 0;
+	k_timeout_t sem_timeout = K_MSEC(10);
```

Comment:
why 10?

Reply:
From what I remember, the caam should have the random numbers generated on the order of microseconds, so 10 milliseconds seemed like a reasonable timeout, but I will admit it's somewhat arbitrary